bigquery_schema_generator/generate_schema.py

```diff
@@ -939,6 +939,7 @@ def bq_schema_to_map(schema):
     'TIME',
     'DATETIME',
     'RECORD',
+    'NUMERIC',
```

Comment:
Thanks for the PR.

When I created `bigquery-schema-generator`, I don't recall seeing a `NUMERIC` or `BIGNUMERIC` type. So I think this was added later. You are touching a part of the code that was contributed, so I might be a bit rusty on the impact of this change. The `generate_schema.py` never creates a `NUMERIC` type. So the only way this would be triggered is if you are using the `--existing_schema_path` flag to pass in an existing schema. Is that right?

Do you happen to have a small sample data that uses this functionality, so that it can be added to the integration tests?

Also, if we are adding `NUMERIC`, it seems like we should also add `BIGNUMERIC` as well.
https://cloud.google.com/bigquery/docs/data-types#numeric-type-support